src/services/item/plugins/app/appData/service.ts

```
@@ -130,6 +142,11 @@ export class AppDataService {
       throw new AppDataNotFound(appDataId);
     }
 
+    // prevent patch on app data file
+    if (currentAppData?.data && currentAppData.data[this.fileItemType]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)